src/offsetPath.js

```diff
@@ -0,0 +1,62 @@
+/* global internalScope */
+
+(function () {
+  function offsetPathParse (input) {
+    var parseAngleAsDegrees = internalScope.parseAngleAsDegrees;
+
+    // https://drafts.fxtf.org/motion-1/#offset-path-property
+
+    if (input === 'none') {
+      return {type: null, input: null};
+    }
+
+    var inputType = '';
+    var ray = /^ray\((.*)\)$/.exec(input);
+    var path = /^path\((.*)\)$/.exec(input);
+
+    if (ray === null && path === null) {
+      return undefined;
+    } else if (ray !== null) {
+      inputType = 'ray';
+      var rayInput = ray[1];
+      var rayInputDegrees = parseAngleAsDegrees(rayInput);
+      if (rayInputDegrees !== null) {
```

Comment:
if (rayInputDegrees === null) {
return undefined;
}

with a test that asserts undefined for ray(pants) or similar.